filters.kak

```diff
@@ -73,7 +73,7 @@ define-command peneira-symbols -docstring %{
 
     peneira 'symbols: ' %{
         export LUA_PATH="$kak_opt_peneira_path/?.lua"
-        env lua=$kak_opt_luar_interpreter "$kak_opt_peneira_path/filters" symbols "$kak_buffile"
+        exec $kak_opt_luar_interpreter "$kak_opt_peneira_path/filters.lua" symbols "$kak_buffile"
```

Comment:
Hi, @natsukagami ! 

Sorry for not testing it beforehand, but those `exec`s seem to break the commands. Do them work for you?
It seems we need to remove them.

Sorry again.

Once that is done, we can merge your changes.

Reply:
Hmm, looking at the whole source code, I think this is making it not work:

https://github.com/gustavo-hms/peneira/blob/429f0422f4395564811d9c73d51a78b772dbd4e4/peneira.kak#L81-L87

If we call `exec` in the `eval` block it will exit when `lua` does and never writes the filename to register... So we can either not use `exec` or do something with this part of the code (maybe move the `echo` line up?). 

I shall just remove the `exec` part, but maybe you would prefer the other way?

Reply:
Removing the `exec` is fine by now. In the future I may explore an alternative, but, by now, we can keep it as is.

Thank you very much!